### DISCUSSION TITLE
VS Code extention not working

### DISCUSSION THREAD

**Discussion by NiranjanGanesh17** [hyperledger-labs/solang#720] [General]
After installing VS code extention i can't get the version of the "solang"  , im unable to use solang

![image](https://user-images.githubusercontent.com/71085177/161069945-86a22ee2-8e0d-4e90-b6f0-de929bda1c91.png)

After this i tried building from the github, im getting the following error 
![image](https://user-images.githubusercontent.com/71085177/161070773-d343ca02-6e6d-4cb6-879e-2b9f6d7f30b5.png)

**Comment by seanyoung** [comment]
@NiranjanGanesh17 if you install the vscode extension, then that does not mean you have solang in your path, iow. available on the command line.

If you want to use solang on the command line then it is best to download it from here: https://github.com/hyperledger-labs/solang/releases/tag/v0.1.10

You need to do `chmod 755 solang-linux-x86-64 ` and then you can it as `./solang-linux-x86-64`.

In order to install solang from the github source, you need to install llvm first, see https://solang.readthedocs.io/en/latest/installing.html#installing-the-llvm-libraries

**Reply by NiranjanGanesh17** [reply]
Thanks @seanyoung for the reply
